manifest.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Something doesn't look quite right, as this manifest.yml file comes into play in the 5th tutorial step.